library/core/src/mem/maybe_uninit.rs

```diff
@@ -64,7 +64,7 @@ use crate::mem::ManuallyDrop;
 /// *immediate* undefined behavior, but will cause undefined behavior with most
 /// safe operations (including dropping it).
 ///
-/// [`Vec<T>`]: ../../std/vec/struct.Vec.html
+/// [`Vec<T>`]: std::vec::Vec<T>
```

Comment:
This is in `core` and `Vec` is in `alloc`. So this link will not resolve: #74481.
```suggestion
/// [`Vec<T>`]: ../../std/vec/struct.Vec.html
```

Reply:
Thanks...I was misunderstanding. I will fix it and squash the commit